projects/e2e-tests/codecov_tests/src/lib.rs

```diff
@@ -281,39 +281,35 @@ pub extern "C" fn finish() -> i32 {
                 amount.len(),
             )
         },
-        19,
+        62,
         "trace_amount",
     );
 
     // ########################################
     // Step #2: Test set_data edge cases
     // ########################################
-    check_result(
-        unsafe { host_bindings_loose::get_parent_ledger_hash(-1, 4) },
-        error_codes::INVALID_PARAMS,
-        "get_parent_ledger_hash_neg_ptr",
-    );
-    with_buffer::<4, _, _>(|ptr, _len| {
-        check_result(
-            unsafe { host_bindings_loose::get_parent_ledger_hash(ptr as i32, -1) },
-            error_codes::INVALID_PARAMS,
-            "get_parent_ledger_hash_neg_len",
-        )
-    });
     with_buffer::<3, _, _>(|ptr, len| {
         check_result(
-            unsafe { host_bindings_loose::get_parent_ledger_hash(ptr as i32, len as i32) },
+            unsafe { host::get_parent_ledger_hash(ptr, len) },
             error_codes::BUFFER_TOO_SMALL,
             "get_parent_ledger_hash_buf_too_small",
         )
     });
-    with_buffer::<4, _, _>(|ptr, _len| {
-        check_result(
-            unsafe { host_bindings_loose::get_parent_ledger_hash(ptr as i32, 1_000_000_000) },
-            error_codes::POINTER_OUT_OF_BOUNDS,
-            "get_parent_ledger_hash_len_too_long",
-        )
-    });
+    // NOTE: This test is commented out because the WASM runtime catches the out of bounds
```

Comment:
LMK and I'll simply remove these commented out tests. Wanted to leave them in for now to make sure we evaluate this change-set properly.